views/Dashboard/TaskDetails.ejs

```diff
@@ -188,9 +188,10 @@
                     >
                   </div>
                   <div class="col-sm-4 col-xl-6 input-column">
-                    <label class="col-form-label" for="name-input-field"
-                      ><%= task.dueDate %></label
-                    >
+                    <label class="col-form-label" for="name-input-field">
+                        <% let longStringDate = task.dueDate.toString() %> 
+                        <%=longStringDate.substring(0, 16) %>
```

Comment:
Does this work with all the dates and months ?

Reply:
@davejay1999 can you confirm ?

Reply:
Yes, it will... hence we change dates into strings and then substring